etc/config-template.toml

```diff
@@ -38,6 +38,18 @@ address = ""
 job = "tikv"
 
 [raftstore]
+# Base tick interval
+# raft-base-tick-interval = 100ms
+
+# Ticks to trigger a hearbeat
+# raft-heartbeat-ticks = 3
+
+# Ticks that must pass between elections. It must be greater than
```

Comment:
For good user experience, docs should also be provided for

1. the brief description of tick, heartbeat, election timeout, etc
2. the principle of how to configure them correctly, pros and cons, example cases

We could mark a TODO here or somewhere in the task management system, to recap the corresponding docs/guide in the future.